builtin/settingtypes.txt

```diff
@@ -891,8 +891,17 @@ default_privs (Default privileges) string interact, shout
 #    Privileges that players with basic_privs can grant
 basic_privs (Basic privileges) string interact, shout
 
-#    If enabled, disable cheat prevention in multiplayer.
-disable_anticheat (Disable anticheat) bool false
+#    Configure server anticheat
+#    Combine the byteflags below to enable desired anticheat features, or set to 0
+#    to disable anticheat:
+#    DIGGING: 1 (Prevent fast/instant digging cheats)
+#    INTERACTION: 2 (Prevent illegal player interactions)
+#    MOVEMENT: 4 (Prevent moving too fast)
+anticheat_flags (Anticheat flags) int 7 0 7
```

Comment:
There's a dedicated `flags` setting type which can be used here.

Reply:
> There's a dedicated `flags` setting type which can be used here.

But `flags` type is not proper for this implementation, since it uses string flags, e.g.
`mgfractal_spflags (Mapgen Fractal specific flags) flags terrain terrain,noterrain`
vs byteflags
`csm_restriction_flags (Client side modding restrictions) int 62 0 63`

For settings GUI yes, string-based flags are more human-understable, but for dedicated servers it's faster to type single digit instead of typing couple of string flags manually

Reply:
Regarding the "server owners" point: If you're editing minetest.conf via ssh and you read `anticheat_flags = 3`, you have to look up `anticheat_flags` in minetest.conf.example before the value becomes useful. If you read `anticheat_flags = digging,interaction,nomovement`, the value is useful right away.